### ISSUE TITLE
Possible security vulnerabilities

### ISSUE THREAD

**Issue by iteles** [cul-2016/quiz#680]
As noted on the homepage of this repository, there are currently possible security vulnerabilities.

@stianr I believe that as per our email exchange you would like this to be reviewed as a priority today?
If these can be updated without much time involved, they should be updated, however if the updates carry breaking changes, we will need your guidance on whether to spend time on this over #678 

We'll begin by investigating.

![screen shot 2018-07-10 at 09 51 14](https://user-images.githubusercontent.com/4185328/42499807-a4558626-8427-11e8-953b-2061f20be42d.png)

**Comment by stianr** [comment]
@iteles yes please. Do you have access to the alerts - I think there are 3-4.

**Comment by sophielevens** [comment]
Having looked into this, I have:
+ upgraded `Hoek`
+ removed `jsdocs` (which was unused)
+ not updated `moment` as it would involve updating HapiJs to version 17 which is a major change
